src/common/math/Vector.h

```diff
@@ -876,17 +876,6 @@ namespace Math {
 		return init;
 	}
 
-	// Vector minimum/maximum and clamping
```

Comment:
Unused I assume?

Reply:
There was one solitary use case of Distance which I moved to the vec3_t API.